src/angular/projects/demo/src/app/app.module.ts

```diff
@@ -0,0 +1,20 @@
+import { NgModule } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+
+import { AppComponent } from './app.component';
+import { PartytownModule } from '../../../partytown/src/lib/partytown.module';
+import { PartytownGtmModule } from '../../../partytown/src/lib/partytown-gtm.module';
```

Comment:
Maybe a path that targets `./partytown/src/lib/*` to avoid relative paths? Otherwise, it's dope, mate! I'm pulling rn 😝

Reply:
Yes 100% this was just a PoC. The paths and structure aren't figured out yet. Working on it!

Reply:
I've also only tested that the scripts are added haven't actually setup the service worker and web worker copy stuff yet!